### DISCUSSION TITLE
Version 1.7.8

### DISCUSSION THREAD

**Discussion by deleolajide** [igniterealtime/openfire-pade-plugin#461] [General]
This is the last version of pade for Openfire 4.7.x 
The next version of pade will be for Openfire 4.8+

<hr /><em>This discussion was created from the release <a href='https://github.com/igniterealtime/openfire-pade-plugin/releases/tag/v1.7.8'>Version 1.7.8</a>.</em>

**Comment by gjaekel** [comment]
Thank you!
